src/core/deadbeef.d

```diff
@@ -0,0 +1,141 @@
+/**
+ * This module contains deadbeef().
+ *
+ * Copyright: D Language Foundation 2019
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Walter Bright
+ * Source:    $(DRUNTIMESRC core/_deadbeef.d)
+ */
+
+module core.deadbeef;
+
+/****************************************
+ * Step on all the register contents that
+ * are not saved by the calling convention
+ * of the function.
+ * Useful for:
+ *
+ * 1. testing the code generator
+ * 2. avoiding leaking information that should be hidden
+ * from the caller
+ *
+ * Inserting a call to this function at any point in the code
+ * should not produce any failures.
+ */
+
+nothrow
+@safe
+@nogc
+pure
+void deadbeef()
```

Comment:
Seems like a pretty poor name, and I don't see the relationship to uses mentioned here:  https://en.wikipedia.org/wiki/Hexspeak
Maybe something like `clobberRegisters` would be more appropriate.

Reply:
`deadbeef` has a long history of what uninitialized things should be set to, so they stand out in the debugger.

Reply:
That's fine, but please change the name of the function to something more appropriate.  You might even consider allowing the caller to pass in a value suitable for their needs, perhaps defaulting to `0xdeadbeef`.  Some may prefer to just use `0`.

Reply:
Using 0 is a poor idea, since 0 is so often a valid value for a register. The reason for `0xdeadbeef` is because it is highly unlikely to be a correct value, because it shows up in the debugger much like a NaN does. I don't see a point to a user-specified value.